### PR TITLE
Feat: Added HomePage

### PR DESCRIPTION
**Pull request by Manpreet-hub**
### Build Home-Page component for E-commerce:

- Added Header Section
- Banner Card
- Recommend Product section
- Footer Section

### Live Preview
chrome_BBNnn2WN0v

### Below are  main files to Check
-  Header.jsx
-  BannerCard.jsx
-  Footer.jsx
-  Home.jsx
-  RecommendProducts.jsx
_P.S :- Ignore other files. They are WIP but you can also comment on those files (if required)._

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

**Review by ishapatell1 — APPROVED** [review]
UI looks Good, we can work more on colours!

File: src/components/homepage/BannerCard.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import "../../styles/components/banner-card.css";
+
+const BannerCard=()=>{
+    return (
+        <>
+        <main className="main-section">
+        <div className="hero-container">
+            <h1 className="title">Fashion-hub Sale</h1>
+            <p>UPTO 50% OFF on various products</p>
+            <a href="/pages/products/products.html" className="shop-btn"><button className="btn-default btn-primary">Shop
```
Comment (ajay-mandaviya) [inline src/components/homepage/BannerCard.jsx:10]: Remove a tag use Link or Navigate insted of a

File: src/components/homepage/RecommendProducts.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import brandArmani from "../../assets/images/armani.jpg";
+import brandGucci from "../../assets/images/gucci.jpg";
+import brandNorthFace from "../../assets/images/northFace.jpg";
+
+import suit from "../../assets/images/suit.jpg";
+import whiteShirt from "../../assets/images/man-white-shirt.jpg";
+import dress from "../../assets/images/dress.jpg";
+import jeans from "../../assets/images/blue-jeans.jpg";
+import tShirt from "../../assets/images/black-t-shirt.jpg";
+
```
Comment (ajay-mandaviya) [inline src/components/homepage/RecommendProducts.jsx]: make a common index.js file  , where you can import all images and you can import here it's a good structure

File: src/components/homepage/RecommendProducts.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <img className="card-img product-img img img-round" src={dress} alt="product-img"/>
+                <p class="p para-md">Dresses</p>
+            </div> 
+
+            <div className="card">
+                <img className="card-img product-img img img-round" src={jeans} alt="product-img"/>
+                <p class="p para-md">Jeans</p>
+            </div> 
+        </div>
+
```
Comment (ajay-mandaviya) [inline src/components/homepage/RecommendProducts.jsx:24]: you can  make array of object and map it here for this images

File: src/components/homepage/RecommendProducts.jsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+            </div>
+            <div className="card">
+                <img className="card-img" src={brandGucci} alt="product-img"/>
+            </div>
+            <div className="card">
+                <img className="card-img" src={brandNorthFace} alt="product-img"/>
+            </div>
+        </div>
+    </div>
+
```
Comment (ajay-mandaviya) [inline src/components/homepage/RecommendProducts.jsx]: same here for make array of object and map it here

**Review by ajay-mandaviya — APPROVED** [review]

File: src/components/homepage/RecommendProducts.jsx (hunk 2 above)
Comment (Manpreet-hub) [reply src/components/homepage/RecommendProducts.jsx]: Done a4a3afd394857327a6495d55074fe08a881ab0a7

File: src/components/homepage/RecommendProducts.jsx (hunk 4 above)
Comment (Manpreet-hub) [reply src/components/homepage/RecommendProducts.jsx]: Done https://github.com/Manpreet-hub/ecommerce-react/commit/a4a3afd394857327a6495d55074fe08a881ab0a7

File: src/components/homepage/BannerCard.jsx (hunk 1 above)
Comment (Manpreet-hub) [reply src/components/homepage/BannerCard.jsx:10]: Thanks!! will add in next PR.

**Pull request merged**